boostlook.css

```diff
@@ -105,54 +108,64 @@ text-decoration: underline;
 .boostlook code {
 font-family: "Noto Sans Mono", monospace;
 }
-.boostlook pre {
+.boostlook:not(.body) pre {
 font-family: "Noto Sans Mono", monospace;
 background-color: #f8f8f8;
 margin: 1em;
-padding: 1em;
+padding: 1rem;
+overflow-x: auto;
 }
 
 .boostlook h6:has(+table)
 {
 margin-left: 1em;
 }
-.boostlook table {
+.boostlook:not(.body) table {
 border-collapse: collapse;
 margin: 1em;
+border: 1px solid #e8e8e8;
 }
-.boostlook div:has(table, pre, code) {
-	overflow-x: scroll;
+
+.boostlook #content p code,
+.boostlook #content p tt,
+.boostlook #content p kbd,
+.boostlook #content p samp,
+.boostlook #content p pre,
+.boostlook .content p code,
+.boostlook .content p tt,
+.boostlook .content p kbd,
+.boostlook .content p samp,
+.boostlook .content p pre {
+    background-color: transparent;
 }
-.boostlook code,
-.boostlook tt,
-.boostlook kbd,
-.boostlook samp,
-.boostlook pre 
-{
-    background: none;
-  }
-.boostlook th {
-text-align: left;
-padding: 0.25em 0.55em;
-font-weight: 550;
+.boostlook:not(.body) th {
+  background-color: #f8f8f8;
+  text-align: left;
+  padding: 0.25em 0.55em;
+  font-weight: 550;
 }
-.boostlook td {
-padding: 0.25em 0.55em;
+.boostlook:not(.body) td {
+  border: 1px solid #e8e8e8;
+  padding: 0.25em 0.55em;
 }
 
+.boostlook [class^="sect"]:not(.sectionbody) pre,
+.boostlook [class^="sect"]:not(.sectionbody) table,
+.boostlook [class^="sect"]:not(.sectionbody) code:not(p code) {
+    overflow-x: auto;
+}
 /*----------------------------------------------*/
 
 :root {
 --color-pasteboard: rgb(209, 213, 219);
 }
 
-.boostlook:not(.body) {
+.article.toc2.toc-left {
```

Comment:
```suggestion
.boostlook .article.toc2.toc-left {
```

Reply:
We're not able to do that since `.boostlook` is inside `.article.toc2.toc-left` for asciidoc

Reply:
actually, since these are formatting issues, they should  probably belong in the docs css, as the boostlook code should not contain layout formatting, which is what that appears to do. boostlook classes should be universal to whichever doc they are being used in.

Reply:
I agree, but the suggestion breaks the layout for asciidoc. The rule has to live here since we're not modifying `asciidoc.css` directly.

Here's the built html: [mp11.html.zip](https://github.com/user-attachments/files/16785997/mp11.html.zip)

Reply:
got it, in that case we should try to have any specific content changes inside a commented block, something like:
```
/*----------------- changes specific to asciidocs -----------------*/

/*-----------------end  changes specific to asciidocs -----------------*/
```

because we want future devs to be aware that these aren't global effects. and where specifically they are used.

Reply:
Love that ❤️. I'll update